doc/samtools-view.1

```diff
@@ -442,6 +446,23 @@ is equivalent to
 .BI "-@ " INT ", --threads " INT
 Number of BAM compression threads to use in addition to main thread [0].
 .TP
+.BR -P ", " --fetch-pairs
+Retrieve pairs even when the mate read is outside of the requested region.
+Enabling this option also turns on the multi-region iterator (\fB-M\fR).
+A region to search must be specified, either on the command-line, or using
+the \fB-L\fR option.
+The input file must be an indexed regular file.
+
+This option first scans the requested region to look for mate pairs outside
+the original search area.
```

Comment:
Perhaps it should be noted that the mate pairs look up is done only for reads passing filters if any were given.

Reply:
Can we use different terminology than "mate pairs"? It's unclear as to whether secondary/supplementary alignment should or should not be included and [mate pair sequencing](https://sapac.illumina.com/science/technology/next-generation-sequencing/mate-pair-sequencing.html) isn't what this PR is referring to.